src/hotspot/share/runtime/arguments.cpp

```diff
@@ -2735,7 +2735,7 @@ jint Arguments::parse_each_vm_init_arg(const JavaVMInitArgs* args, bool* patch_m
         return JNI_EINVAL;
       }
 
-      if (FLAG_SET_CMDLINE(MaxTenuringThreshold, max_tenuring_thresh) != JVMFlag::SUCCESS) {
+      if (FLAG_SET_CMDLINE(MaxTenuringThreshold, (uint)max_tenuring_thresh) != JVMFlag::SUCCESS) {
```

Comment:
MaxTenuringThreshold was recently changed to uint, so it would make sense change uintx above to uint.

Reply:
Isn't parse_integer() used to parse int flags?  It's a template.

Reply:
Ok, I fixed parse_uintx in Arguments to parse_uint then.